server/cluster/cluster.go

```diff
@@ -1196,8 +1196,11 @@ func (c *RaftCluster) deleteStoreLocked(store *core.StoreInfo) error {
 func (c *RaftCluster) collectMetrics() {
 	statsMap := statistics.NewStoreStatisticsMap(c.opt)
 	stores := c.GetStores()
-	for _, s := range stores {
-		statsMap.Observe(s, c.storesStats)
+	for _, store := range stores {
+		if store.IsTombstone() || store.IsUnhealth() {
```

Comment:
how we got the unhealth count?  `statsMap` including: https://github.com/tikv/pd/blob/master/server/statistics/store_collection.go#L67-L86